vuepress/docs/next/tutorials/consume/mt-cds.md

```diff
@@ -0,0 +1,285 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: YOUR-CDS-DEPLOYMENT
+  namespace: YOUR-NAMESPACE
+  labels:
+    app: YOUR-CDS-DEPLOYMENT
+spec:
+  selector:
+    matchLabels:
+      app: YOUR-CDS-DEPLOYMENT
+  template:
+    metadata:
+      labels:
+        app: YOUR-CDS-DEPLOYMENT
+    spec:
+      containers:
+        - readinessProbe:
+            httpGet:
+              port: 8081
+              path: /health/health_check
+              scheme: HTTP
+            failureThreshold: 1
+            initialDelaySeconds: 5
+            periodSeconds: 5
+            successThreshold: 1
+            timeoutSeconds: 5
+          env:
+            - name: RUST_LOG
+              value: actix_web=info,actix_server=info,actix_web_middleware_keycloak_auth=trace
+            - name: KEYCLOAK_PUBLIC_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: KC_PUBLIC_KEY
+                  name: YOUR-PUBLIC-KEY
+            - name: CORS_ALLOWED_ORIGIN # use for external CDS service
+              value: All # enter your Entando app domain name
+            - name: CORS_ALLOWED_ORIGIN_END_WITH # use for wildcard domain name
+              value: "$CORS" # enter wildcard domain name for your Entando app, e.g. *domain.com
+          name: cds
+          image: docker.io/entando/cds:1.0.4
+          imagePullPolicy: IfNotPresent
+          livenessProbe:
+            httpGet:
+              scheme: HTTP
+              port: 8081
+              path: /health/health_check
+            timeoutSeconds: 5
+            successThreshold: 1
+            periodSeconds: 30
+            initialDelaySeconds: 5
+            failureThreshold: 1
+          ports:
+            - containerPort: 8080
+              name: internal-port
+            - containerPort: 8081
+              name: public-port
+          resources:
+            limits:
+              cpu: 500m
+              memory: 500Mi
+            requests:
+              cpu: 500m
+              memory: 500Mi
+          volumeMounts:
+            - mountPath: /entando-data
+              name: cds-data-volume
+      volumes:
+        - name: cds-data-volume
+          persistentVolumeClaim:
+            claimName: YOUR-PVC-CLAIM
+            readOnly: false
+  replicas: 1
+```
+
+### Ingress Descriptor
+``` yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: YOUR-INGRESS-NAME
+  namespace: YOUR-NAMSPACE
+  annotations:
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
+      proxy_set_header X-Scheme \$scheme;
+      proxy_set_header X-Forwarded-Proto \$scheme;
+      add_header Content-Security-Policy upgrade-insecure-requests;
+    nginx.ingress.kubernetes.io/proxy-body-size: "150m" # edit according to the file size you require
+
+    nginx.org/client-max-body-size: "150m" # edit according to the file size you require
+spec:
+  $INGRESS_CLASS_FIELD
```

Comment:
I guess this should be not here, it might be `ingressClassName: nginx` instead.

Reply:
It causes an error
`error: error parsing cds-ingress-descriptor.yaml: error converting YAML to JSON: yaml: line 17: mapping values are not allowed in this context`